bsoid_app/data_preprocess.py

```diff
@@ -10,216 +10,175 @@
 from bsoid_app.bsoid_utilities.likelihoodprocessing import *
 from bsoid_app.bsoid_utilities.load_json import *
 
+software_choice = os.environ.get('software_choice', '')
```

Comment:
This is a wrong list, but I'd move this section to `bsoid_app.py` and modify this class (and its name) to accept these variables as inputs during class initialization similar to `Extract`. So we can contain the environment variable craziness.